### PR TITLE
K8 Envoy logging custom format walkthrough

### PR DESCRIPTION
**Pull request by paulyeo21**
*Issue #, if available:*

N/A

*Description of changes:*

EKS envoy custom access logging format walkthrough

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by EdwardXF — APPROVED** [review]

File: walkthroughs/howto-k8s-envoy-logging-custom-format/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+## Overview
+This example shows how http routes can use headers for matching incoming requests.
```
Comment (bendu) [inline walkthroughs/howto-k8s-envoy-logging-custom-format/README.md:2]: Should we say this example is for custom logging format instead?

File: walkthroughs/howto-k8s-envoy-logging-custom-format/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+4. **(Optional) Specify Envoy Image version** If you'd like to use a different Envoy image version than the [default](https://github.com/aws/eks-charts/tree/master/stable/appmesh-controller#configuration), run `helm upgrade` to override the `sidecar.image.repository` and `sidecar.image.tag` fields.
+
+5. Deploy
+    ```
+    ./deploy.sh
+    ```
+
+## Test
+
+Port-forward pod for simulationg
```
Comment (bendu) [inline walkthroughs/howto-k8s-envoy-logging-custom-format/README.md:37]: Spelling?

**Review by bendu — CHANGES_REQUESTED** [review]

File: walkthroughs/howto-k8s-envoy-logging-custom-format/README.md (hunk 1 above)
Comment (paulyeo21) [reply walkthroughs/howto-k8s-envoy-logging-custom-format/README.md:2]: yeup thanks

File: walkthroughs/howto-k8s-envoy-logging-custom-format/README.md (hunk 2 above)
Comment (paulyeo21) [reply walkthroughs/howto-k8s-envoy-logging-custom-format/README.md:37]: yeup thanks again

**Review by bendu — APPROVED** [review]

**Pull request merged**